<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInfo</class>
 <widget class="QDialog" name="DialogInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informations</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelFirstUse">
         <property name="text">
          <string>First use date and time</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditFirstUse"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelDatabaseLocation">
         <property name="text">
          <string>Database location</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditDatabaseLocation"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelNumberOfGamesOfDatabase">
         <property name="text">
          <string>Number of games in database</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditNumberOfGame"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelSizeOfDataBase">
         <property name="text">
          <string>Size of database in byte</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditSizeOfDatabase"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
